crates/jstz_node/src/node_runner.rs

```diff
@@ -0,0 +1,51 @@
+use std::io;
+
+use actix_web::{middleware::Logger, web::Data, App, HttpServer};
+use env_logger::Env;
+use octez::OctezRollupClient;
+use tokio_util::sync::CancellationToken;
+
+use crate::services::{AccountsService, LogsService, OperationsService, Service};
+
+/// Endpoint defailts for the `jstz-node`
+const ENDPOINT: (&str, u16) = ("127.0.0.1", 8933);
+
+pub async fn run_node(
+    rollup_node_rpc_addr: String,
+    rollup_node_rpc_port: u16,
+    rollup_endpoint: Option<String>,
+    kernel_file_path: String,
+) -> io::Result<()> {
+    env_logger::init_from_env(Env::default().default_filter_or("info"));
```

Comment:
We may want to disable the logger in the sandbox, wdyt?

Reply:
I've added an option to disable it.

What is the actual purpose of it?